samples/core/src/main/java/io/oasp/gastronomy/restaurant/offermanagement/service/impl/rest/OffermanagementRestServiceImpl.java

```diff
@@ -142,8 +135,7 @@ public OfferEto updateOffer(OfferEto offer) {
    * @param product the product to save
    * @return the saved product
    */
-  @POST
-  @Path("/product/")
+  @Override
```

Comment:
Just a simple example for a general issue: after you extracted the interface you should remove the javadocs from the implementation to avoid redundancies... Please do this for all methods and not just this one.